server/server.lua

```diff
@@ -67,6 +67,9 @@ end)
 
 RegisterNetEvent('qb-scenes:server:CreateScene', function(sceneData)
     local src = source 
+    local Player = QBCore.Functions.GetPlayer(src)
+        
+    TriggerEvent("qb-log:server:CreateLog", "scenes", "New Scene Created", "red", "**".. GetPlayerName(source) .. "** (citizenid: *"..Player.PlayerData.citizenid.."* | id: *"..source.."*) created a new scene; scene: **"..sceneData.text.."**, where: **" .. sceneData.coords .. "**")
 
     MySQL.Async.insert('INSERT INTO scenes (creator, text, color, viewdistance, expiration, fontsize, fontstyle, coords, date_creation, date_deletion) VALUES (? ,?, ?, ?, ?, ?, ?, ?, NOW(), DATE_ADD(NOW(), INTERVAL ? HOUR))', {
         QBCore.Functions.GetPlayer(src).PlayerData.citizenid, 
```

Comment:
```suggestion
        Player.PlayerData.citizenid, 
```
Only thing I would change since we're adding a local variable for Player. Also what does this log look like when triggered? Curious about how the coords are shown as table or as a vector, FiveM down or I'd check it myself.

Reply:
So the current look is this, however, I plan on further adapting it to be a bit more suitable. 
![image](https://user-images.githubusercontent.com/15137702/161411661-ed5d453b-2703-45ef-9823-aeacfc3da0a9.png)

Also, good call on the suggested change above. I feel like I attempted that and it wasn't playing nice, but it has been a bit now.